examples/acquia/acquia.config.yml

```diff
@@ -4,38 +4,6 @@ vagrant_box: geerlingguy/ubuntu1204
 
 # Acquia Cloud defaults to PHP 5.6.
 php_version: "5.6"
-php_install_recommends: no
```

Comment:
Reminder to self—I'll need to update BLT's internal config.yml once this stuff lands.